apps/web/src/components/Messages/MessageHeader.tsx

```diff
@@ -46,21 +55,43 @@ const MessageHeader: FC<MessageHeaderProps> = ({ profile }) => {
           onClick={onBackClick}
           className="mr-1 h-6 w-6 cursor-pointer lg:hidden"
         />
-        <UserProfile profile={profile} />
+        {profile ? (
+          <UserProfile profile={profile} />
+        ) : (
+          <>
+            <Image
+              onError={({ currentTarget }) => {
+                currentTarget.src = getAvatar(profile, false);
+              }}
+              src={getAvatar(profile)}
+              loading="lazy"
+              className="mr-4 h-10 w-10 rounded-full border bg-gray-200 dark:border-gray-700"
+              height={40}
+              width={40}
+              alt={formatHandle('')}
+            />
+            {ensNames.get(conversationKey ?? '') ??
+              shortAddress(conversationKey ?? '')}
+          </>
+        )}
       </div>
-      {!following ? (
-        <Follow
-          showText
-          profile={profile}
-          setFollowing={setFollowingWrapped}
-          followSource={FollowSource.DIRECT_MESSAGE_HEADER}
-        />
-      ) : (
-        <Unfollow
-          showText
-          profile={profile}
-          setFollowing={setFollowingWrapped}
-        />
+      {profile && (
+        <div>
```

Comment:
this can just be a fragment here

Reply:
Because there is no HTML child immediate that the dom can find, es lint throws an error, saying you don't need a fragment here, till there is at least one child